app/src/main/java/ingsw/group1/findmyphone/location/LocationMessageParser.java

```diff
@@ -0,0 +1,105 @@
+package ingsw.group1.findmyphone.location;
+
+import android.location.Location;
+
+/**
+ * Parser class of static methods that provides composition of message for location request and response
+ * Message request is composed as:
+ * {@link LocationMessageParser#LOCATION_REQUEST_TAG}
+ * <p>
+ * Message response is composed in this way:
+ * {@link LocationMessageParser#LOCATION_RESPONSE_TAG}<LT>value_latitude</LT><LG>value_longitude</LG>
+ *
+ * @author Giorgia Bortoletti
+ */
+public class LocationMessageParser {
+
+    static final String LOCATION_REQUEST_TAG = "LOCATION_REQUEST";
+    static final String LOCATION_RESPONSE_TAG = "LOCATION_RESPONSE";
+    static final String LONGITUDE_TAG = "<LG>", LONGITUDE_END_TAG = "</LG>";
+    static final String LATITUDE_TAG = "<LT>", LATITUDE_END_TAG = "</LT>";
+
+    //---------------------------- COMPOSE MESSAGE ----------------------------
+
+    /**
+     * Compose a location request message
+     *
+     * @return a formatted message for a location request
+     */
+    static String composeRequestLocation() {
+        return LOCATION_REQUEST_TAG;
+    }
+
+    /**
+     * Return a String formatted as a message containing actual position
+     *
+     * @param foundLocation the found location the device needs to send back
+     * @return a formatted string containing the location as <>longitude</> <>latitude</>
+     */
+    static String composeResponseLocation(Location foundLocation) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/RicDeZen/ingsw-group1-findmyphone/pullRequest?prid=4929934)